cpp/memilio/math/adapt_rk.cpp

```diff
@@ -73,6 +73,10 @@ Tableau::Tableau()
 bool RKIntegratorCore::step(const DerivFunction& f, Eigen::Ref<const Eigen::VectorXd> yt, double& t, double& dt,
                             Eigen::Ref<Eigen::VectorXd> ytp1) const
 {
+    assert(0 <= m_dt_min);
+    assert(m_dt_min < m_dt_max);
```

Comment:
should m_dt_min = m_dt_max also be possible if we want to force a fixed step size?

Reply:
Yes, good catch. This should be <=